src/kernel/filesystem/vfs.zig

```diff
@@ -218,6 +218,17 @@ pub const DirNode = struct {
         }
         return fs.open(fs, node, name, flags, args);
     }
+
+    /// See the documentation for FileSystem.Close
+    pub fn close(self: *const DirNode) void {
```

Comment:
As with `FileNode.close`, we should make the close function be part of the `Node` struct instead.